### PR TITLE
More french strings

### PR DESCRIPTION
**Pull request by erickgirard**
Translated group and guidelines sections 😃

### CODE REVIEW

File: locales/fr.yml [hunk 1]
```diff
@@ -101,41 +101,41 @@ fr:
...
+      rules_placeholder: Quels sont les rêgles du groupe?
+      submit_button_text: Créer mon nouveau groupe
+      terms_and_conditions_community_guidelines_link: Rêgles de la communauté
+      terms_and_conditions_group_leader_guidelines_link: Lignes directrices des gestionnaires de groupe
+      terms_and_conditions_terms_of_use_link: Conditions d'utilisation
+      title_description: Un groupe est l'endroit où les kinksters peuvent se réunir pour partager leurs expériences, leurs intérêts communs, leurs projets et plus encore!
+      title_title: Créer un nouveau groupe
   guidelines:
-    header: FetLife Guidelines & Legalese
+    header: Directives de FetLife & Blabla Légal
```
Comment (nielsenramon) [inline locales/fr.yml]: I would leave out the blabla part and just keep legal?

File: locales/fr.yml [hunk 2]
```diff
@@ -101,41 +101,41 @@ fr:
...
       guidelines:
-        description: To help make FetLife a fun place for everyone, we kindly ask everyone to follow our Content, Community, and Group guidelines.
-        title: Guidelines
+        description: Pour aider à faire de FetLife un endroit amusant pour tous, nous vous demandons de bien vouloir suivre nos consignes relatives au contenu, à la communauté et au groupe.
+        title: Lignes directrices
       legalese:
-        description: This section contains the Terms of Use all members agree to when they sign up for FetLife. We've tried to keep things as simple and clear as possible!
-        title: Legalese
+        description: Cette section contient les conditions d'utilisation que tous les membres acceptent lors de leur inscription à FetLife. Nous avons essayé de garder les choses aussi simples et claires que possible!
+        title: Blabla légal
```
Comment (nielsenramon) [inline locales/fr.yml]: Same here I would leave out Blabla!

**Comment by nielsenramon** [comment]
@erickgirard Awesome thanks! Just 2 minor comments and we can get this merged!

**Comment by erickgirard** [comment]
Good point, I was uncertain since the "bla bla" part was used elsewhere, there is no direct translation in french for legalese (i.e. the slang expression for legal talk) I'll change this up tonight 🙂

**Comment by erickgirard** [comment]
There, removed present and previous references to "blabla" 😎👍

**Comment by nielsenramon** [comment]
@erickgirard Thanks a lot! 🚀

**Pull request merged**